api.yaml

```diff
@@ -95,7 +95,7 @@ info:
     | Status code   |                          Message                            |
     |---------------|-------------------------------------------------------------|
     |         `403` | Forbidden - Missing or incorrect API key                    |
-    |         `405` | Method Not Allowed - Incorrect method was used (`GET/POST`) |
+    |         `405` | Method Not Allowed - Incorrect HTTP method was used         |
```

Comment:
**nitpick (non-blocking)**: we need to update also the [mock server](https://github.com/Tiqets/supplier-api/blob/3df0850cc5cae7201f6560431e9a69ed3bf8fb8a/supplier_server_mock/supplier_server/error_handlers.py)